scripts/regression-qemu.sh

```diff
@@ -11,11 +11,17 @@ compatibility_tests=(
   "dasharo-compatibility/custom-boot-menu-key.robot"
   "dasharo-compatibility/uefi-shell.robot"
   "dasharo-compatibility/network-boot.robot"
+  "dasharo-compatibility/dmidecode.robot"
+  # FIXME: unsafe to enable https://github.com/Dasharo/dasharo-issues/issues/887
+  #"dasharo-compatibility/reset-to-defaults.robot"
 )
 
 security_tests=(
   "dasharo-security/network-stack.robot"
   "dasharo-security/secure-boot.robot"
+  # TODO: test if this is safe to enable
```

Comment:
I hope we can remove the regression-qemu script and use a single regression.sh for all platforms. But I will take it later: https://github.com/Dasharo/open-source-firmware-validation/issues/320

Reply:
This reminded me that I've tested it and forgot to remove `TODO`.